docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -0,0 +1,78 @@
+= Restore a PostgreSQL service
+
+IMPORTANT: The restores can only be triggered from an instance in the same namespace.
+If the old instance has already been deleted the backups are deleted as well.
+
+. Get the name of the backup you want to restore by listing all available backups.
++
+[source,bash]
+----
+kubectl -n postgres get vshnpostgresbackups
+BACKUP NAME                                 DATABASE INSTANCE     STATUS     AGE
```

Comment:
I'd expect a printer column from when this backup actually is. Yes, it's in the name, but that's not very user-friendly. Could we expose `status.process.timing.stored` here?

Reply:
Yes we can do that.